lib/pill.hoon

```diff
@@ -0,0 +1,122 @@
+::  |pill: helper functions for making pills
+::
+|%
+::  +module-ova: vane load operations.
+::
+::    sys: full path to /sys directory
+::
+++  module-ova
+  |=  sys=path
+  ^-  (list ovum)
+  |^  :~  ::  sys/zuse: standard library
```

Comment:
This looks like it should just be a +turn.

Reply:
Sure, that seems appropriate. `|^` is usually kind of weird.